schema.prisma

```diff
@@ -522,6 +525,48 @@ model BankTransactionsFile {
   @@map("bank_transactions_files")
 }
 
+model CampaignReport {
+  id                  String               @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  campaignId          String               @map("campaign_id") @db.Uuid
+  campaign            Campaign             @relation(fields: [campaignId], references: [id])
+  startDate           DateTime             @map("start_date")
+  endDate             DateTime             @map("end_date")
+  description         String
+  files               CampaignReportFile[]
+  totalFunds          Float?               @map("total_funds")
+  fundsForPeriod      Float?               @map("funds_for_period")
+  spentFundsForPeriod Float?               @map("spent_funds_for_period")
```

Comment:
@batebobo Normally we use only `Integer` types for storing the amounts, so I would suggest that we keep that practice here as well to be consistent with the other use cases.

Stripe also provides the amounts in the smallest currency value.

Since the columns are optional we might add a default value of `0` so we always have a value.

```suggestion
  totalFunds          Int?              @map("total_funds") @default(0)
  fundsForPeriod      Int?              @map("funds_for_period") @default(0)
  spentFundsForPeriod Int?              @map("spent_funds_for_period") @default(0)
```

Reply:
Yeah, makes sense, thanks :) I'll update the schema :v: